config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
+
+  # Defines the root path route ("/")
+  # root "posts#index"
+end
+  root "users#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
+end
```

Comment:
- [ ] [OPTIONAL] You've done a great job so far but I noticed an error when running the command `rails routes | grep user` command and the issue is from the `end` keyword you have at line `10` as it doesn't correspond to any block and is redundant so I'd suggest you remove it.
  ![image](https://github.com/Young-Gaius/Rails-Blog-App/assets/104968236/5fce4c23-bb7c-438f-ab21-a2e0ce0400d0)